plugins/inputs/internet_speed/internet_speed.go

```diff
@@ -73,12 +74,14 @@ func (is *InternetSpeed) Gather(acc telegraf.Accumulator) error {
 		"download": is.server.DLSpeed,
 		"upload":   is.server.ULSpeed,
 		"latency":  timeDurationMillisecondToFloat64(is.server.Latency),
+		"jitter":   timeDurationMillisecondToFloat64(is.server.Jitter),
 	}
 	tags := map[string]string{
 		"server_id": is.server.ID,
 		"host":      is.server.Host,
 	}
-
+	// recycle the detailed data of each test to prevent data backlog
+	is.server.Context.Reset()
```

Comment:
What is the data backlog you refer to here?

Reply:
I am referring to the history data of each request inside speedtest-go, If it is not cleaned for a long time, it will take up more and more memory.